x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/field_mapping_row.tsx

```diff
@@ -4,48 +4,66 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { useState } from 'react';
-import { EuiFlexItem, EuiFlexGroup, EuiSuperSelect, EuiIcon } from '@elastic/eui';
+import React from 'react';
+import {
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiSuperSelect,
+  EuiIcon,
+  EuiSuperSelectOption,
+} from '@elastic/eui';
 
+import { capitalize } from 'lodash/fp';
 import * as i18n from './translations';
+import {
+  CaseField,
+  ActionType,
+  ThirdPartyField,
+} from '../../../../containers/case/configure/types';
 
-interface ThirdPartyField {
-  value: string;
-  inputDisplay: JSX.Element;
-}
 interface RowProps {
-  siemField: string;
-  thirdPartyOptions: ThirdPartyField[];
+  disabled: boolean;
+  siemField: CaseField;
+  thirdPartyOptions: Array<EuiSuperSelectOption<ThirdPartyField>>;
+  onChangeActionType: (caseField: CaseField, newActionType: ActionType) => void;
+  onChangeThirdParty: (caseField: CaseField, newThirdPartyField: ThirdPartyField) => void;
+  selectedActionType: ActionType;
+  selectedThirdParty: ThirdPartyField;
 }
 
-const editUpdateOptions = [
+const actionTypeOptions: Array<EuiSuperSelectOption<ActionType>> = [
   {
     value: 'nothing',
-    inputDisplay: <span>{i18n.FIELD_MAPPING_EDIT_NOTHING}</span>,
+    inputDisplay: <>{i18n.FIELD_MAPPING_EDIT_NOTHING}</>,
```

Comment:
do you need to wrap this in `<>{}</>` or can it just be `inputDisplay: i18n.FIELD_MAPPING_EDIT_NOTHING,`?

Reply:
I think so. `inputDisplay` requires a `JSX.Element`